src/commands/migrateQiskitCode.ts

```diff
@@ -51,74 +179,45 @@ async function handler(): Promise<void> {
   }
 
   try {
-    const notificationTitle = `Migrating the ${selection.isEmpty ? "document" : "selected"} code`;
-
-    let end = lastLine.range.end;
-    let migratedText = await vscode.window.withProgress({
-      location: vscode.ProgressLocation.Notification,
-      cancellable: false,
-      title: notificationTitle
-    }, async (progress):Promise<string> => {
+    setLoadingStatus('connecting');
+    
+    const migrationResult = await (async (): Promise<MigrationResult | null> => {
+      setLoadingStatus('generating');
       
-      progress.report({  increment: 10, message: "Please wait..." });
-
-      let t = ""
-      let responseData = migrateCode(text);
-      let step = 0;
-      for await (let chunk of responseData) {
-        if ((chunk as unknown as {error: string})?.error) {
-          throw Error((chunk as unknown as {error: string})?.error)
-        }
+      let result = await migrateCode(text);
 
-        // update notidication message based on streaming data progress
-        if (chunk.plan_steps && step < 1) {
-          progress.report({  message: "Planning migration...", increment: 25 });
-          step = 1;
-        }
-        if (chunk.final_thoughts && step < 2) {
-          progress.report({  message: "Reviewing migration...", increment: 25 });
-          step = 2;
-        }
-        if (chunk.migrated_code && step < 3) {
-          progress.report({  message: "Returning response...", increment: 25 });
-          step = 3;
-        }
+      setLoadingStatus('processing');
```

Comment:
it looks like the code that was written to [support streaming](https://github.com/Qiskit/qiskit-code-assistant-vscode/pull/118/commits/63e47515790c017c4e4e6206d420acb7a7e04d4c) has been removed? is there a specific reason for this or was it in inadvertent?

Reply:
streaming of migration is no longer working. can you see about restoring this code?